src/server/api/lib/middleware/admin/admin.js

```diff
@@ -0,0 +1,11 @@
+const admin = require('firebase-admin');
```

Comment:
I suggest changing the name to firebase so that its not confused with the role

Reply:
@senner007 I have made the changes based on your feedback. Please review :)